pkg/ingester/ingester_test.go

```diff
@@ -4219,6 +4221,8 @@ func getWALReplayConcurrencyFromTSDBHeadOptions(userTSDB *userTSDB) int {
 }
 
 func TestIngester_shipBlocks(t *testing.T) {
+	ctx := context.Background()
```

Comment:
shipBlocks further down uses context.Backnground(), should we change it to use ctx now?

Reply:
Thanks, done.